src/lib/strategy/metadata.ts

```diff
@@ -8,49 +8,57 @@ import type { StrategyConfiguration } from './configuration';
 import { assert } from 'assert-ts';
 import loadError from '../assets/load-error.jpg';
 
+type Nullable<Type> = Type | null;
+type PerformanceTuple = [number, number];
+
 /**
- * Metadata describes strategy information not related to the profit generation.
+ * StrategySummaryStatistics describes summary-level performance metrics for a strategy.
  *
- * This is bits like name, description, icon and executor uptime.
+ * See https://github.com/tradingstrategy-ai/trade-executor/blob/master/tradeexecutor/strategy/summary.py
+ */
+export interface StrategySummaryStatistics {
+	calculated_at: number;
+	first_trade_at: Nullable<number>;
+	last_trade_at: Nullable<number>;
+	enough_data: Nullable<boolean>;
+	current_value: Nullable<number>;
+	profitability_90_days: Nullable<number>;
+	performance_chart_90_days: Nullable<PerformanceTuple[]>;
+}
+
+/**
+ * Metadata describes strategy information not related to the profit generation.
  *
- * TypeScript helper for having frontend side configuration for strategies.
+ * This includes properties provided by the backend API, augmented with additional properties
+ * client-side (e.g., link, config, error).
  *
  * See https://github.com/tradingstrategy-ai/trade-executor/blob/master/tradeexecutor/state/metadata.py
  */
 export interface StrategyMetadata {
```

Comment:
Let's call this `StrategyRuntimeState` - calling it metadata is misleading, because it now contains other than metadata after these changes

Reply:
👌 got it – I will rename the file and helper methods as well, then.

For consistency, do you think the backend endpoint should be renamed from `metadata` to `runtime-state`?

Reply:
(per Discord conversation … yes, but later)

Reply:
@miohtama this change is done!